src/goto-symex/memory_model.cpp

```diff
@@ -74,18 +74,12 @@ void memory_model_baset::read_from(symex_target_equationt &equation)
       }
 
       // value equals the one of some write
-      exprt rf_some;
+      exprt rf_some = big_or(rf_some_operands);
```

Comment:
1) There is `disjunction` in `std_expr.h`, which you could use instead of introducing `big_or`.
2) I'd now move this assignment and the comment below the next block as `rf_some` will not be needed if the `continue` is hit.

Reply:
I did (1) but wrt. (2) there is not `continue` in the latest version. Is it ok?

Reply:
Indeed, but you could still move it into the body of the if statement?

Reply:
You mean in that particular commit? Because in the latest version, it's already there.

Reply:
Ah, my apologies, I had missed that. Perfectly fine as-is now.